eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -1,5 +1,67 @@
 <UsageData>
   <IgnorePatterns>
-    <UsagePattern IdentityGlob="*/*" />
+    <UsagePattern IdentityGlob="Microsoft.SourceBuild.Intermediate.*/*" />
+    
+    <!-- TODO: Ignore needed until https://github.com/NuGet/Home/issues/11059 is addressed. -->
+    <UsagePattern IdentityGlob="Nuget.*/*" />
+    <UsagePattern IdentityGlob="Microsoft.Build.NuGetSdkResolver/*" />
+    
+    <!-- TODO: Ignore needed until vstest gets a source build leg. -->
+    <UsagePattern IdentityGlob="Microsoft.TestPlatform.*/*" />
```

Comment:
vstest does have a source-build leg today.  Do you know the underlying issue?  Are they not publishing their intermediate correctly?  An issue should get logged to track this work.

Reply:
They're not on arcade publishing so the intermediate doesn't go anywhere.  However, https://github.com/microsoft/vstest/pull/4332 is their full onboarding to arcade, which will bring in the SB validation, with publishing included.

Reply:
Would be nice to reference that work in the comment IMO.